effekt/shared/src/main/scala/effekt/machine/Tree.scala

```diff
@@ -217,7 +215,7 @@ export Type.{ Positive, Negative }
 type Evidence = Int
 object builtins {
 
-  val Evidence = Type.Int()
+  val Evidence = Type.Extern("%Int 8")
```

Comment:
Does it make sense to define this in the prelude as well?

```
extern type Evidence ="..."
```

Reply:
I am unsure about that one, too.
On one hand, we do not currently have a mechanism to hide it from the user in that case. On the other hand, that might be introduced via #30 someday, and this is very ad hoc.

Is the representation of evidence sufficiently uniform? If so, we could in principle also maybe have `LiftInference` introduce it (as some kind of type alias or something like that).

Reply:
I think the latter is a great idea! We also need to document somewhere which builtin types are required to exist when.